uwp/Ribbon/BackStage.md

```diff
@@ -76,9 +76,9 @@ Ribbon_backstage.Items.Add(Exit_backstagebutton)
 
 {% endtabs %}
 
-![BackStage_img1](BackStage_images/BackStage_img1.jpeg)
+![BackStage commands in UWP Ribbon](backStage_images/uwp-ribbon-backstage-commands.jpeg)
```

Comment:
Change image folder name stil have the capital letter.

Reply:
Check the changes in all places.

Reply:
made  changes